starknet_py/abi/model.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Dict, Optional, OrderedDict
+
+from starknet_py.cairo.data_types import CairoType, StructType
+
+
+@dataclass
+class Abi:
+    """
+    Dataclass representing class abi. Contains parsed functions, events and structures.
+    """
+
+    @dataclass
+    class Function:
+        """
+        Dataclass representing function's abi.
+        """
+
+        name: str
+        inputs: OrderedDict[str, CairoType]
+        outputs: OrderedDict[str, CairoType]
+
+    @dataclass
+    class Event:
+        """
+        Dataclass representing event's abi.
+        """
+
+        name: str
+        data: OrderedDict[str, CairoType]
+
+    defined_structures: Dict[str, StructType]  #: Structures' abi defined by the class.
```

Comment:
```suggestion
    defined_structures: Dict[str, StructType]  #: Abi of structures defined by the class.
```
I had to read it 3 times at first 🙈